controls/srg_gpos/SRG-OS-000478-GPOS-00223.yml

```diff
@@ -6,7 +6,7 @@ controls:
     the following: to provision digital signatures, to generate cryptographic hashes,
     and to protect unclassified information requiring confidentiality and cryptographic
     protection in accordance with applicable federal laws, Executive Orders, directives,
-    policies, regulations, and standards.'
```

Comment:
What was the reasoning for removing the Oxford comma?

Reply:
Hum, in other languages I am aware, it is grammatically incorrect to insert a comma before `and` in the last element of a enumeration. So, it was a natural reaction from my side. : ) However, after your comment, I researched the nuances of the grammar and learned that this comma is actually optional in English, for this case. Thus, I will respect the author's choice. : ) I have to update this PR soon, then I revert this change. Thanks for pointing this.